src/layout/NestedPicker.js

```diff
@@ -28,16 +28,20 @@ let FilteredSection = observer(
   )
 )
 FilteredSection.displayName = 'FilteredSection'
-let Section = observer(({ options, onClick, selected, Item }) => (
+let Section = observer(({ options, onClick, selected, Item = 'div' }) => (
   <div>
     {F.mapIndexed(
       (item, key) => (
         <Item
           key={key}
           onClick={() => onClick(item.value || key, item)}
-          active={selected === key}
           disabled={selected && selected !== key}
-          hasChildren={!isField(item)}
+          {...(Item === 'div'
```

Comment:
I think this is a little more straightforward if you just check if `Item` exists and if not ```
<div onClick="" key="">..</div>```. There is some small duplication, but the code is ultimately a little easier to read IMO.

Reply:
Is that what you had in mind? I personally like `const extraProps = ...` and then `<Item {...extraProps}>` more

Reply:
Sorry my force-pushing made this comment break since the commit hashes are not valid anymore. See https://github.com/smartprocure/contexture-react/pull/137/files#diff-9408d26625840440e038edcea38b6d24R34